docs/usage_guide.rst

```diff
@@ -1311,6 +1286,78 @@ For deposits you can use the following filters:
 .. [1] Suggestions will appear for this field based on the available data.
 .. [2] The date filter has to be in the DD/MM/YYYY HH:mm:ss format. You can completely skip the time part or just the seconds part, thus making DD/MM/YYYY or DD/MM/YYYY HH:mm acceptable.
 
+
+
+History events
```

Comment:
why move those?

Reply:
So I move this under "Historical Events" header, because I feel it's easier to find.
Previously it was under "Tracking accounts and balances"

<img width="308" alt="image" src="https://github.com/rotki/rotki/assets/26648140/83196db5-4ade-4eee-a90c-fc211b82f1a0">